app/modal/edit-local-model/[modelId].tsx

```diff
@@ -102,7 +102,7 @@ export default function EditLocalModelScreen() {
             />
           </View>
           <View style={styles.textFieldSection}>
-            <Text style={styles.label}>Model URL</Text>
+            <Text style={styles.label}>Model file</Text>
```

Comment:
```suggestion
            <Text style={styles.label}>Model File</Text>
```

Let's do `file` uppercase, I think it looks better, also in other places we are doing it like that, e.g `Model Name`